#!/usr/bin/env python
PACKAGE = "tuw_iwos_odometer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("revolute_velocity_tolerance", double_t, 0, "tolerance for revolute velocity matching",  0.01, 0.0, 1.0)
gen.add("steering_position_tolerance", double_t, 0, "tolerance for steering position matching", 0.01, 0.0, 1.0)
gen.add("calculation_iterations", int_t, 0, "number of odometer calculation iterations", 1000, 0, 1000000)

exit(gen.generate(PACKAGE, "tuw_iwos_odometer", "JointStateOdometer"))
